src/Microsoft.PowerShell.ConsoleHost/resources/CommandLineParameterParserStrings.resx

```diff
@@ -273,6 +273,6 @@ EXAMPLES
     <value>Invalid argument '{0}', did you mean:</value>
   </data>
   <data name="DeprecatedVersionParameter" xml:space="preserve">
-    <value>Usage of '-Version {0}' is not supported. '-Version' currently only returns the current PowerShell version.</value>
+    <value>Unsupported value '{0}' passed to `-Version`. '-Version' only returns the current PowerShell version and must be used standalone.</value>
```

Comment:
I still unlike this:
1. `-Version`. '-Version'  - looks bad.
2. Unsupported value - implies that there are supported values.

Reply:
If we follow the example of some other tools like: git, curl, bash (although each of them use `--version`), they basically ignore anything after `--version` and simply output the version string.  Perhaps that's the right behavior.

Reply:
For now, I think I'll follow the other tools and will update the documentation as well.

Reply:
Initial idea for PowerShell was to inform users that we change -Version semantics. So I write information error and PowerShell version too.